src/ui/components/radioSelector/RadioSelector.web.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { fireEvent, render, screen } from 'tests/utils/web'
+
+import { RadioSelector } from './RadioSelector'
+
+describe('<RadioSelector />', () => {
+  it('should call press when pressing Spacebar', () => {
+    const onPress = jest.fn()
+    render(<RadioSelector label="label" onPress={onPress} />)
+
+    const Press = screen.getByTestId('label')
+    fireEvent.focus(Press)
+    fireEvent.keyDown(Press, { key: 'Spacebar' })
+
+    expect(onPress).toBeCalledTimes(1)
```

Comment:
Comme on l'a évoqué tout à l'heure, c'est bien de tester que ça a été appelé 1 fois, c'est mieux de tester que ça n'a pas appelé avec un argument.
```suggestion

    expect(onPress).toHaveBeenCalledNthTimesWith(1)
```